### PR TITLE
Rename BaseLog::resolve() to interpolate()

### PR DESCRIPTION
**Pull request by othercorey**
Let's use a more consistent name before this is finalized.

Perhaps we can replace it with a general interpolation helper in the future.

### CODE REVIEW

**Comment by ADmad** [comment]
Nice, `resolve()` did sound weird for what it does.

File: src/Log/Engine/BaseLog.php [hunk 1]
```diff
@@ -121,21 +121,21 @@ public function scopes()
      * @param string $message The message to be formatted.
      * @param array $context Additional logging information for the message.
      * @return string
-     * @deprecated 4.3.0 Call `resolve()` directly from your log engine and format the message in a formatter.
+     * @deprecated 4.3.0 Call `interplate()` directly from your log engine and format the message in a formatter.
      */
     protected function _format(string $message, array $context = []): string
     {
-        return $this->resolve($message, $context);
+        return $this->interplate($message, $context);
```
Comment (ADmad) [inline src/Log/Engine/BaseLog.php]:
```suggestion
        return $this->interpolate($message, $context);
```

Comment (othercorey) [reply src/Log/Engine/BaseLog.php]: yea I saw this too.

**Comment by othercorey** [comment]
> Nice, `resolve()` did sound weird for what it does.

I remember thinking it special for some reason, but re-reading standards, I was wrong.

**Pull request merged**